internal/service/identity_api/request.go

```diff
@@ -0,0 +1,31 @@
+package identity_api
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/DIMO-Network/telemetry-api/internal/config"
+	"github.com/Khan/genqlient/graphql"
+)
+
+type IdentityService interface {
+	AftermarketDevice(ctx context.Context, by AftermarketDeviceBy) (*aftermarketDeviceResponse, error)
+}
+
+type identityAPI struct {
+	client graphql.Client
+}
+
+func NewIdentityService(settings *config.Settings) IdentityService {
```

Comment:
Don't return interfaces. Just make the IdentyAPI public and return the struct if a package needs an interface they can define it, but this pattern is forcing all consumers to add the complexity of interfaces even if they don't need it.
https://go.dev/wiki/CodeReviewComments#interfaces